.github/workflows/deploytoAksCluster.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
```

Comment:
master or main?

Reply:
master is version/tag of checkout action. Its not related to our github branch.